src/manipulation.js

```diff
@@ -33,9 +33,27 @@ var
 		option: [ 1, "<select multiple='multiple'>", "</select>" ],
 
 		thead: [ 1, "<table>", "</table>" ],
-		col: [ 2, "<table><colgroup>", "</colgroup></table>" ],
-		tr: [ 2, "<table><tbody>", "</tbody></table>" ],
-		td: [ 3, "<table><tbody><tr>", "</tr></tbody></table>" ],
+
+		// Following wrappers are not fully defined because
+		// their parent elements (except for "table" element) could be omitted
+		// since browser parsers are smart enough to auto-insert them
+		col: [
+
+			2,
+			// Support: Android 2.3
+			// Android browser doesn't auto-insert colgroup element
+			"<table><colgroup>", "</colgroup></table>"
+		],
+
+		tr: [
+			2, // - auto-inserts "tbody" element
```

Comment:
I think you misunderstood. Assuming `auto-inserts "tbody" element` and `auto-inserts "tbody" and "tr" elements` are not also limited to Android 2.3, I would expect something like this:
```js
thead: [ 1, "<table>", "</table>" ],

// Browsers auto-insert tbody
tr: [ 2, "<table>", "</table>" ],

// Browsers auto-insert tbody and tr
td: [ 3, "<table>", "</table>" ],

// Support: Android <=2.3 only
// Android browser doesn't auto-insert colgroup
col: [ 2, "<table><colgroup>", "</colgroup></table>" ],
```